collections/editor/skeletalsubmit.php

```diff
@@ -109,15 +110,21 @@
 							<input name="addaction" type="radio" value="2" /> <?php echo $LANG['APPEND_VALUES']; ?>
 						</fieldset>
 					</div>
-					<div style="position:absolute;background-color:white;top:10px;right:10px;">
+					<!-- <div style="position:absolute;background-color:white;top:10px;right:10px;">
```

Comment:
If we need to comment something out, I'd prefer to delete it unless there's a really good reason to leave it in.